src/PrestaShopBundle/Entity/Repository/NormalizeFieldTrait.php

```diff
@@ -63,16 +63,22 @@ protected function castIdsToArray($rows)
     }
 
     /**
-     * @param $columnName
+     * @param String $columnName
+     * @param String $columnValue
+     * 
      * @return bool
      */
-    private function shouldCastToInt($columnName)
+    private function shouldCastToInt($columnName, $columnValue)
     {
-        return false !== strpos($columnName, '_id') ||
-        false !== strpos($columnName, 'id_') ||
-        false !== strpos($columnName, '_quantity') ||
-        false !== strpos($columnName, 'sign') ||
-        false !== strpos($columnName, 'active') ||
-        false !== strpos($columnName, 'total_');
+        $isConcernedColumn = false !== strpos($columnName, '_id') ||
+            false !== strpos($columnName, 'id_') ||
+            false !== strpos($columnName, '_quantity') ||
+            false !== strpos($columnName, 'sign') ||
+            false !== strpos($columnName, 'active') ||
+            false !== strpos($columnName, 'total_') ||
+            false !== strpos($columnName, 'product_low_stock_threshold') ||
+            false !== strpos($columnName, 'low_stock_alert');
+
+        return $isConcernedColumn && !is_null($columnValue) && 'N/A' !== $columnValue;
```

Comment:
You should perform these two checks first
```php
null !== $columnValue && 'N/A' !== $columnValue;
```
And the other checks should be handled by a regex with boundary checks (cf. "[Clbuttic mistake](https://blog.codinghorror.com/obscenity-filters-bad-idea-or-incredibly-intercoursing-bad-idea/)")